packages/flame_spine/example/LICENSE

```diff
@@ -0,0 +1,24 @@
+flame_spine_example License
```

Comment:
I don't think this license is needed, as `example/` is not a standalone package that will be published -- instead, it's just a part of the root `flame_spine` package, so the LICENSE of `flame_spine` covers the example/ too.